docs/kb/kb0011-configuration-migration.md

```diff
@@ -0,0 +1,152 @@
+    ```
+
+    **Note**: `/path/to/` will differ on your local system. Please update this to be the local respective path.
+
+1. The script will launch and prepare to connect. You will be prompted for a username and password which will obtain a JWToken if successful. Once connect, it will attempt to export all the desired configuration specified above.
+
+    Depending on how much configuration there is to export, this may take up to 15 minutes to complete. Once export has completed, move onto the import phase.
+
+##### Importing configuration (Destination system)
+
+1. Open up a new `PowerShell` session to avoid any conflict with environmental variables
+1. In the `PowerShell` window, run the following:
+
+    **Note**: We will be using the ficticious front end `https://cluedin-destination.customer.com` in the example below.
+
+    ```powershell
+    $BaseURL = 'customer.com'
+    $Organization = 'cluedin-destination'
+    $RestorePath = '/path/to/export-20240403'
+
+    /path/to/CluedIn.Product.Toolkit/Scripts/Import-CluedInConfig.ps1 \
+        -BaseURL ${BaseURL} \
+        -Organization ${Organization} \
+        -RestorePath ${$RestorePath}
+    ```
+
+    **Note**: `/path/to/` will differ on your local system. Please update this to be the local respective path.
+
+1. The script will launch and prepare to connect. When successful, it will iterate through the RestorePath and begin importing and/or correcting any configuration drift between the environments. 
+
+    The console will output any relevant changes that may have occurred during the import phase so that you can keep track of what has happened.
+
+    {:.important}
+    A lot of the comparisons are based on `Display Name` lookup. If a display name matches and there is configuration drift, it will attempt to correct this during import time.
+    It will not delete any configuration as that is not within scope.
+
+    Depending on how much configuration there is to import, this may take up to 15 minutes to complete.
+
+This concludes the general process of how to export and import configuration from one environment to another.
+
+#### Azure DevOps (Pipeline)
+
+This guide will be using the `import` and `export` scripts only. The toolkit also supports running the functions on their own, but this will not be covered as part of this documentation.
+
+To successfully run this, you will need the following information before proceeding:
+- Full url of your frontend
+- Access to Azure DevOps with the pipelines setup (Please refer to the README within the `CluedIn Product Toolkit`). Failing to setup the pipelines correctly will result in failed configuration backups and restores.
+- Id's of the data you want to backup
+
+{:.important}
+To get the Ids of specific parameters, you will need to use the frontend and navigate to the desired resource. In the url, you will then notice the id of a given object. This may be a guid or integer depending on what is viewed.
```

Comment:
```suggestion
To get the IDs of specific parameters, you will need to use the front end and navigate to the desired resource. In the URL, you will then notice the ID of a given object. This may be a GUID or integer depending on what is viewed.
```